packages/runtime/src/runtime.ts

```diff
@@ -127,26 +140,6 @@ export class Runtime {
     return account.getLocalState(appId, key);
   }
 
-  /**
-   * Add new key-value pair or updating pair with existing key in
-   * app's global data for application id: appId, throw error otherwise
-   * @param appId: current application id
-   * @param key: key to fetch value of from local state
-   * @param value: key to fetch value of from local state
-   */
-  setGlobalState (appId: number, key: Uint8Array | string, value: StackElem): Map<string, StackElem> {
```

Comment:
Should we remove this function from runtime? Is it possible that during testing user wants to set global state explicitly before calling the smart contract?

Reply:
I don't think that's ideal, User will be testing their TEAL files, not our code. If that makes sense

Reply:
It just seems to me we're taking control away from the user. Eg For a stateful counter application, the user can use `runtime.setGlobalState('counter', 5)` explicitly, and then call the smart contract to assert if counter has incremented by 1 or not. What's the benefit of removing it? 
Anyway, if you will remove it, please update the `/docs` as well. It is mentioned there user can use `setGlobalState`.

Reply:
fair enough, restoring setGlobalState